README.md

```diff
@@ -1,86 +1,91 @@
 
 ## Dependencies
-This tool depends on:
+ParaStell depends on:
 
-- [The VMEC tools](https://github.com/aaroncbader/pystell_uw) developed by @aaroncbader 
+- [PyYAML](https://pyyaml.org/wiki/PyYAMLDocumentation)
+- [NumPy](https://numpy.org/install/)
+- [SciPy](https://scipy.org/install/)
+- [scikit-learn](https://scikit-learn.org/stable/install.html)
+- [Coreform Cubit](https://coreform.com/products/downloads/), version 2023.11
 - [CadQuery](https://cadquery.readthedocs.io/en/latest/installation.html)
 - [MOAB](https://bitbucket.org/fathomteam/moab/src/master/)
-- [Coreform Cubit](https://coreform.com/products/downloads/) or [Cubit](https://cubit.sandia.gov/downloads/) by Sandia National Laboratories
-- [cad-to-dagmc](https://pypi.org/project/cad-to-dagmc/)
-- [Gmsh](https://pypi.org/project/gmsh/)
-- [YAML](https://pyyaml.org/wiki/PyYAML)
-## Install with Mamba
+- [PyStell-UW](https://github.com/aaroncbader/pystell_uw) developed by @aaroncbader 
+- [CAD-to-DAGMC](https://github.com/fusion-energy/cad_to_dagmc)
 
-This guide will use the mamba package manager to install dependencies in a conda environment. Conda allows for easily installing and switching between different versions of software packages through the use of [environments](https://conda.io/projects/conda/en/latest/user-guide/concepts/environments.html).
+## Install Python Dependencies
 
-If you have not already installed conda, you can use one of the following installers to do so: [Miniconda](https://docs.conda.io/en/latest/miniconda.html), [Anaconda](https://www.anaconda.com/), or [Miniforge](https://github.com/conda-forge/miniforge).
+This guide will use the conda and mamba package managers to install Python dependencies. Conda provides straight-forward installation of Python packages and switching between different collections of Python packages through the use of [environments](https://conda.io/projects/conda/en/latest/user-guide/concepts/environments.html).
 
-Mamba is available through conda with `conda install -c conda-forge mamba`. Begin by creating a new conda environment with mamba installed.
+If you have not already installed conda, you can use one of the following installers to do so:
+- [Miniforge](https://github.com/conda-forge/miniforge)
+- [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
+- [Anaconda](https://www.anaconda.com/)
 
-```
-conda create -n parastell_env
-conda activate parastell_env
-conda install -c conda-forge mamba
-```
+Mamba is a faster, more reliable conda alternative that is fully compatible with conda packages. Mamba is available via conda (note that Miniforge ships with mamba already installed).
 
-The subsequent mamba and pip install commands should be run with this environment activated.
+### Option 1: Using the ParaStell Environment File
```

Comment:
What are your thoughts on only offering a preferred path using the `environment` on the README, and then referring people to another `INSTALL` doc for more details?  We could then streamline the info on the README to be a simple standard approach.

Reply:
That would be fine with me. Would the `INSTALL` document just be another Markdown file we refer users to? I'm not sure if an `INSTALL` document refers to some standard format.

Reply:
Just another markdown file, `INSTALL.md`.  You could start with this one and then whittle down the `README`